src/Input/AbstractParamAwareInput.php

```diff
@@ -154,9 +154,10 @@ public function getOptions(): array
         return $this->input->getOptions();
     }
 
-    public function isInteractive(): ?bool
+    public function isInteractive(): bool
```

Comment:
Discussed in review below: this being `@internal` (and changing) is alright, and therefore does not constitute a BC break.